### DISCUSSION TITLE
Hero13 LABS issue! not flash this!

### DISCUSSION THREAD

**Discussion by IvanhoeFPV** [gopro/labs#1061] [Q&A]
Attention! the new LABS for Hero13 has a bug. It loops while flashing and cannot complete the flash. After restarting and removing the card, it boots into LABS and immediately freezes and reboots over and over again.

**Comment by dnewman-gpsw** [comment]
What version did you flash the firmware from?  I've had one report from a prerelease camera, that has now been restored and is now running Labs.  Many others are installing without issue.

For others: install and get 1.10 running first, do a factory reset, then install 1.10.70.  This is only a precaution.

**Reply by IvanhoeFPV** [reply]
Yes original update after first launch about 1.10. and then LABS 1.10.70 and there was a problem. Now I managed to do a factory reset and LABS 1.10.70 remained in the camera and the camera is now working.

**Reply by aneedham26** [reply]
Just tried to install labs and I am having the same problem. It just reboots over and over and I am not able to get to preferences to reset to factory. Is there another way to reset it?

**Reply by aneedham26** [reply]
Oh boy...so I just tried to update back to 1.10.70. manually, and now its just doing a reboot loop saying updating. Not sure where to go from here

**Reply by IvanhoeFPV** [reply]
yes the same problem. It helped me to remove the card and quickly switch to photo mode after booting and then I had enough time to go to the factory reset menu.

**Reply by IvanhoeFPV** [reply]
But after a factory reset, LABS stayed in the camera and worked. At some specific setting, the camera froze again and restarted. I put the LABS away and installed the original firmware. LABS has bugs in it.

**Reply by aneedham26** [reply]
Ivanhoe.....Thank you for that!! Switching to photo mode was the trick. Labs didnt stay in mine for some reason though. I'll stick with the GoPro firmware for now. Thanks again!

**Reply by IvanhoeFPV** [reply]
Great news! I was happy to help ;)

**Reply by ardenjacoby** [reply]
Had the same problem. Will have to switch back to gopro firmware till this is fixed

**Comment by DemonViewLLC** [comment]
Just received my Hero 13 and the very first thing I did was install the GoPro labs firmware.

Have the same issue, when I try to set up new video modes, the camera freezes and then reboots.  

Let me know if there's any info I can provide to help you trouble shoot the issue.

**Reply by IvanhoeFPV** [reply]
they say you have to do a factory reset before installing LABS and then it works but I don't know, it seems wrong to me

**Reply by dnewman-gpsw** [reply]
We have confirmed the creation of presets via the menu is flaky under Labs.  It will fixed shortly.  You can create presets via  QR codes

**Reply by RikshaDriver** [reply]
Alternative is to create a new preset via editing an existing preset and tap "Save As" on bottom. This will allow you to create a new preset and edit it accordingly. Just don't modify your existing preset first.

**Comment by toroe0815** [comment]
I had the same problem with boot loops.(solved by factory reset after installing labs firmware).  
Furthermore editing the presets in "Quik"-app does not work!

**Comment by mikkihugo** [comment]
My H13  with labs is rebooting nonstop and no time to get into any menu for a factory reset. Often ends in pink screen. Any suggestion?

**Reply by dnewman-gpsw** [reply]
Add file factory_reset.txt to sd card. This is the other way to do it.
